packages/@react-native-windows/codegen/src/Cli.ts

```diff
@@ -151,31 +162,70 @@ function generate(
 */
 
   const generateNM2 = createNM2Generator({namespace: argv.namespace});
+  const generatorPropsH = require('react-native-tscodegen/lib/rncodegen/src/generators/components/GeneratePropsH')
```

Comment:
FYI that we're still on an older version of tscodegen/codegen. Zihan published a new version, with some possible changes, but I hadn't pulled it into RNW yet.

Reply:
yeah.  I keep wondering if we are going to need to pull tscodegen into rnw repo.  Right now it seems to be ok with the older version.  but it does worry me.

Do we have RN-core publishing the codegen pacakge in daily builds?

Reply:
Nope. Original plan was for me to hookup nightlies for monorepo packages, but code stopped being pulled into separate packages so the urgency dropped off. There is a ready-made tscodegen that is newer than what we have rn though.